Header header
int8 channels
uint16[] lengths
uint16[] tan_force
uint16[] norm_force
uint16[] x_positions
uint16[] intensity
